app/models.py

```diff
@@ -2261,6 +2267,7 @@ def serialize(self):
             'template_id': str(self.template_id),
             'template_version': self.template_version,
             'template_name': self.template.name,
+            'template_content': self.content,
```

Comment:
Does this make sense to be `template_content` rather than `content`? `template_content` to me is a template that hasn't had its placeholders filled in whereas content is where it would have (which is what has happened above)

Reply:
I think this comment still stands?

Reply:
Changed in 57473ef6